cmd/internal/cli/inspect_linux.go

```diff
@@ -40,22 +43,27 @@ var (
 	listApps    bool
 )
 
-type inspectAttributes struct {
-	Apps        string            `json:"apps"`
+type inspectMetadata struct {
+	Apps        string            `json:"apps,omitempty"`
+	AppLabels   string            `json:"apps-labels,omitempty"`
 	Labels      map[string]string `json:"labels,omitempty"`
 	Deffile     string            `json:"deffile,omitempty"`
 	Runscript   string            `json:"runscript,omitempty"`
 	Test        string            `json:"test,omitempty"`
-	Environment map[string]string `json:"environment,omitempty"`
+	Environment string            `json:"environment,omitempty"`
 	Helpfile    string            `json:"helpfile,omitempty"`
 }
 
+type inspectAttributesData struct {
```

Comment:
Its used in `inspectFormat`. I can probably clean up the names here a bit.

Reply:
It is used only there. So you introduced a new type just instead of using `Attributes inspectMetadata `. Don't see a reason for that.

Reply:
Ohh, thats because we want the `attributes` section in the `data` section. Unless you know of a simpler way?... Check out this issue -> https://github.com/sylabs/singularity/issues/2790.

Reply:
You can embed a type and still have a json tag for it